packages/project/src/db/props.ts

```diff
@@ -1,22 +1,75 @@
 import {
   type Tree,
   type AllUserProps,
-  UserProps,
+  type UserProp,
+  // TODO: move zod schema into project package
+  zUserUserProps,
+  zUserDbProps,
 } from "@webstudio-is/react-sdk";
 import { applyPatches, type Patch } from "immer";
 import { prisma } from "@webstudio-is/prisma-client";
+import { formatAsset } from "@webstudio-is/asset-uploader/server";
 
 export const loadByTreeId = async (treeId: Tree["id"]) => {
   const instancePropsEntries = await prisma.instanceProps.findMany({
     where: { treeId },
   });
 
+  // Fin all asset_id in all props
+  const assets_id: string[] = [];
```

Comment:
oh nooo, now snake landed in code and I had to create an issue for another eslint rule https://github.com/webstudio-is/webstudio-designer/issues/445 which is now yours 🙃